data/json/terrain.json

```diff
@@ -278,6 +291,7 @@
         "type" : "terrain",
         "id" : "t_strconc_floor_halfway",
         "name": "half-built reinforced concrete floor",
+        "description": "Unfinished series of rebar and poured concrete, the floor hasn't been smoothed and the roof isn't quite filled in yet.",
```

Comment:
The comma is here being used to separate two distinct clauses, should be a period (preferably), a semicolon or a hyphen.

> Unfinished series of rebar and poured concrete. The floor hasn't been smoothed and the roof isn't quite filled in yet.